activity5/obtain_cluster_information.py

```diff
@@ -0,0 +1,76 @@
+
+# coding: utf-8
+
+# In[1]:
+
+# load models
+import sys
+sys.path.append('../')
+from database.models import Incidencia, AccidentDBScan, DBSCANCluster, get_db_session, NumericCluster
+from sqlalchemy import func
+
+
+# In[2]:
+
+
+def obtain_characteristic(session, cluster, item, conditions):
+    ''' 
```

Comment:
Place the comment right before the method starts, also add comments to [4] and [5]